src/mono/mono/mini/mini.h

```diff
@@ -1308,10 +1308,10 @@ typedef struct {
 	MonoJumpInfo    *patch_info;
 	MonoJitInfo     *jit_info;
 	MonoJitDynamicMethodInfo *dynamic_info;
-	guint            num_bblocks, max_block_num;
-	guint            locals_start;
-	guint            num_varinfo; /* used items in varinfo */
-	guint            varinfo_count; /* total storage in varinfo */
+	int              num_bblocks, max_block_num;
```

Comment:
Is mono moving away from the "g" prefix types? I'm looking at the `gint` below and curious if this is intentional.

Reply:
We traditionally used the g types except gint, which is just int, but there are some inconsistencies in usage. guint is 'unsigned int', but it's easier to type.